### PR TITLE
Issue #14914: Added a new section 4.8.5.4 Field Annotations in google style guide implementation

### PR DESCRIPTION
**Pull request by Zopsss**
issue #14914 

added a new section `4.8.5.4 Field Annotations` in google style guide coverage page and wrote test cases for it

### CODE REVIEW

**Comment by Zopsss** [comment]
@romani don't know why this is failing: https://dev.azure.com/romanivanovjr/romanivanovjr/_build/results?buildId=21710&view=logs&j=d72e04f6-d294-5bfe-b3cd-ffcb523f40f9&t=a021e897-39dd-596d-375e-f941387b47d5

**Comment by Zopsss** [comment]
GitHub, generate site

**Comment by romani** [comment]
looks like Azure system failure to start container. 
restarted

File: src/it/resources/com/google/checkstyle/test/chapter4formatting/rule4854fieldannotations/InputFieldAnnotations.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
+package com.google.checkstyle.test.chapter4formatting.rule4854fieldannotations;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class InputFieldAnnotations {
+    public @interface SomeAnnotation1 { }
```
Comment (romani) [inline src/it/resources/com/google/checkstyle/test/chapter4formatting/rule4854fieldannotations/InputFieldAnnotations.java:7]: please add few methods in this Input to make it clear that behavior on fields is different,

**Review by romani — CHANGES_REQUESTED** [review]
item

File: src/it/resources/com/google/checkstyle/test/chapter4formatting/rule4854fieldannotations/InputFieldAnnotations.java (hunk 1 above)
Comment (Zopsss) [reply src/it/resources/com/google/checkstyle/test/chapter4formatting/rule4854fieldannotations/InputFieldAnnotations.java:7]: done

https://github.com/checkstyle/checkstyle/blob/43e4d77faad75a1e80686f55b5e35c1622cc541a/src/it/resources/com/google/checkstyle/test/chapter4formatting/rule4854fieldannotations/InputFieldAnnotations.java#L45-L53

File: src/it/resources/com/google/checkstyle/test/chapter4formatting/rule4854fieldannotations/InputFieldAnnotations.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+     */
+    @SomeAnnotation1
+    @SomeAnnotation2
+    String name = "Zops";
+
+    /**
+     * testing.
+     */
+    @SomeAnnotation1 @SomeAnnotation2
+    int age = 19;
```
Comment (rdiachenko) [inline src/it/resources/com/google/checkstyle/test/chapter4formatting/rule4854fieldannotations/InputFieldAnnotations.java:26]: Please add a usecase with two annotations, one is on its own line and the second one is on the same line as the field, e.g.:
```
    /**
      * testing.
      */
     @SomeAnnotation1
     @SomeAnnotation2 int age = 19;
```

Comment (Zopsss) [reply src/it/resources/com/google/checkstyle/test/chapter4formatting/rule4854fieldannotations/InputFieldAnnotations.java:26]: done

https://github.com/checkstyle/checkstyle/blob/11ec1da697889f3b08cf7e2cd8f30b593dacc45b/src/it/resources/com/google/checkstyle/test/chapter4formatting/rule4854fieldannotations/InputFieldAnnotations.java#L28-L38

**Comment by Zopsss** [comment]
used new verifyWIthGooglexxxx method in latest push

**Review by romani — APPROVED** [review]
ok to merge

**Comment by romani** [comment]
@Zopsss , please resolve conflict.

**Comment by Zopsss** [comment]
> @Zopsss , please resolve conflict.

done. 

Some CI tests got cancelled but it does not concern the PR

**Comment by romani** [comment]
Restarted.

**Review by rdiachenko — APPROVED** [review]
lgtm

**Pull request merged**